tarpc/src/trace.rs

```diff
@@ -47,7 +47,10 @@ pub struct Context {
 /// A 128-bit UUID identifying a trace. All spans caused by the same originating span share the
 /// same trace ID.
 #[derive(Debug, Default, PartialEq, Eq, Hash, Clone, Copy)]
-#[cfg_attr(feature = "serde1", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(
```

Comment:
I'd prefer to just have one serde impl instead of adding a feature. I'm ok making a breaking change.